src/fulcrum-schema.js

```diff
@@ -69,17 +70,20 @@ instance.compareFormSchemas = (oldForm, newForm, options = {}) => {
     let oldSchema = null;
     let newSchema = null;
 
-    const schema = {
+    const schemas = {
       v1: FormSchemaV1,
-      v2: FormSchemaV2
-    }[options.version];
+      v2: FormSchemaV2,
+      v3: FormSchemaV3
+    };
 
     if (oldForm) {
-      oldSchema = new Schema(oldForm, schema, null);
+      const columns = schemas[options.version || oldForm.schema_version];
+      oldSchema = new Schema(oldForm, columns, null);
```

Comment:
Is it correct that each form's schema could now be loaded with different versions if the version isn't specified?

Reply:
Yes . . . sort of. Right now, _everyone_ passes `options.version` with mobile _always_ using "v1" and Fulcrum always passing "v2". In a future Fulcrum ticket, we're going to:

- Add a non-null `schema_version` column to the `forms` table, with a default of 'v2'
- Stop passing `optinos.version` from Fulcrum

This will allow us to pass in old and new schemas with different `schema_version` values. 

Does that answer the question?

Reply:
Yup, good enough for me, bearing in mind I have essentially 0 expertise in this code :)